sagemaker-pipelines/tabular/data_preparation_with_sagemaker_pipeline_and_data_wrangler/sagemaker-pipelines-preprocess-with-data-wrangler.ipynb

```diff
@@ -0,0 +1,416 @@
+{
```

Comment:
<p><code>SageMaker Pipelines</code> (plural) rather than <code>Pipeline</code> for all uses when referring to the feature title.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>